packages/manager/modules/hub/src/components/user-panel/shortcuts/template.html

```diff
@@ -0,0 +1,20 @@
+<div class="manager-hub-shortcuts">
+    <strong data-translate="hub_user_panel_shortcuts_title"> </strong>
+    <div
+        class="d-flex mt-4"
+        data-ng-repeat="row in $ctrl.shortcuts track by $index"
```

Comment:
```suggestion
        data-ng-repeat="shortcut in $ctrl.shortcuts track by $index"
```
but it will be redundant with https://github.com/ovh/manager/pull/2225/files#diff-ddcddc26288494a4777a60824b6ac274R9